pages/strategy.md

```diff
@@ -0,0 +1,58 @@
+
+### Improve documentation and onboarding for mentors
+
+Over 50% of people who sign up to mentor do not go on to mentor a single solution. I recently emailed all of those who had done this. Nearly 1/3 replied with various explanations, but a clear underlying theme is that getting a mentor to give feedback on their first solution quickly and with enough hand-holding would dramatically increase the amount of people who try mentoring. Providing guides and information that they help upskill those mentors and give them a further safety net would significantly increase their retention.
+
+### Empower our mentors into a self-supporting community
+
+Those mentors who mentor the most regularly are also the most active on our Slack community. We believe this is because they feel the most connected to Exercism, the most supported, and because mentoring becomes more than just replying to students - it becomes about sharing in bigger goal with other like-minded people. It is clear that Slack is also a bad choice of tool for many people as it has a barrier-to-entry, clashes with many people's main work-tool, and is a relatively synchronous experience. It also has no reminder/pull-backs if someone closes the window. If we can find a more asynchronous alternative system where people can be part of the community and the conversation, without needing to be "always on", and where we can remind them of conversations they'd like to see, then we will likely improve the quality, frequency, and inclusiveness of the conversation. Slack is also not a scalable option - having 100,000 mentors in one Slack room does not sound like fun.
+
+### Provide realistic expectations to students
+
+A key frustration for students is that some Tracks have a response time of hours and others have a response time of weeks. This discrepancy means that we frustrate and lose many students, many of whom could eventually become mentors. We also end up with lots of "abandoned" solutions that we don't know are abandoned until the student doesn't respond to the mentor - something that disheartens mentors and causes them to leave. 
+
+By providing realistic response-time expectations for the students in the UI **before** they put the effort into solving a solution, we can remove this frustration for both students and mentors.
+
+### Remove the barrier to mentoring
+
+We currently have a deliberate barrier to mentoring with a wall that you have to sign up through. We have put this in place because we want to try and ensure a level of quality control and therefore want to gate-keep and ensure that people who are mentoring feel like they are up to the taks. However, as we add automation, documentation, and a more integrated community, we will be able to remove this barrier and making mentoring more of "something for everyone" on Exercism, rather than something for the elite.
```

Comment:
> We currently have a deliberate barrier to mentoring with a wall that you have to sign up through.

How about:

_We currently have a deliberate barrier to mentoring, requiring mentors to sign up._

> we want to try and ensure

_try **to** ensure_ I think?

> feel like they are up to the taks

_feel like they are up to the **task**

> rather than something for the elite.

Maybe: _rather than something exclusive._